llama-index-integrations/llms/llama-index-llms-bedrock/llama_index/llms/bedrock/utils.py

```diff
@@ -147,7 +147,9 @@ def get_text_from_stream_response(self, response: dict) -> str:
             return ""
 
     def get_text_from_response(self, response: dict) -> str:
-        return response["content"][0]["text"]
+        if response["content"]:
```

Comment:
I guess it's an Optional type?

nit (if this also works): `return response["content"][0]["text"] or ""`

Reply:
No it does not. It was my earlier implementation, when testing privately.

the `content` array may be empty, and index of list 0 will give  'list index out of range'.

Reply:
Got it. Thanks for the reply.